src/Sarif/HashData.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
```

Comment:
In VS, you can right-click in the using sections and 'sort and remove unused namespaces'. Also, on seeing a grayed out namespace, you can invoke the auto-fixer and resolve this condition across the entire solution. +1 on this change, btw, nice work.